qiskit/visualization/__init__.py

```diff
@@ -127,11 +127,13 @@
 from qiskit.visualization.transition_visualization import visualize_transition
 from qiskit.visualization.array import array_to_latex
 
-from .circuit_visualization import circuit_drawer, HAS_PIL, HAS_PDFLATEX, HAS_PDFTOCAIRO
+# NOTE (Jake Lishman, 2022-01-12): for backwards compatibility. Deprecate these paths in Terra 0.21.
```

Comment:
Is there a reason to not deprecate these now?

Reply:
Wow, something is still using `qiskit.tools.visualization` we made that move like 3 years ago. :)